oid4vc-manager/tests/oid4vp/implicit.rs

```
@@ -171,23 +143,46 @@ async fn test_implicit_flow() {
     .unwrap();
 
     // Create a verifiable presentation using the JWT.
-    let verifiable_presentation =
+    let verifiable_presentation_jwt =
         Presentation::builder(subject_did.parse().unwrap(), identity_core::common::Object::new())
             .credential(Jwt::from(jwt))
             .build()
             .unwrap();
 
-    let verifiable_presentation_input = PresentationInputType::Presentation(Box::new(verifiable_presentation));
+    let verifiable_presentation_jwt = VerifiablePresentationJwt::builder()
+        .iss(subject_did.clone())
+        .sub(subject_did)
+        .aud("my-client-id".to_string())
```

If you're going to use hardcoded data for tests in multiple places I recommend creating consts for them. E.g. TEST_CLIENT_ID = "my-client-id". This improves readability and makes it very clear where the same hardcoded data is being used. And of course saves you quite some time and possibly errors when you need to update this.